Editor/src/Program.bf

```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace SteelEditor
+{
+	class Program
+	{
+		public static int Main(String[] args)
+		{
+			Console.WriteLine("Hello Steel Editor user!");
+
+			return 0;
+		}
+	}
+}

```

Comment:
I can remove this if needed. It was just convenient to be able to actually run the `BasicSteelGame` project.

Reply:
You don't need the editor to run the example project? At least should not need to.

Reply:
My Beef IDE was complaining about not being able to find the main of the Editor when it wasn't a startup project. So I had to make the stub class.